src/views/version.rs

```diff
@@ -38,12 +38,12 @@ pub fn version_button(app: &Piggui) -> Element<Message> {
         border_radius: 4.0,
     };
     Button::new(version_text)
-        .on_press(if !app.showing_toast {
+        .on_press(if !app.toast_handler.showing_toast {
```

Comment:
I would move this into toast handler - so that the state variables for toasts (e.g. showing_toast) are private and no one else needs to know how it works.... e.g. here the Button would just sends a message requesting a Toast, and toast handler handles the logic...and shows it or not...